x-pack/plugins/security_solution/server/search_strategy/security_solution/factory/hosts/details/helpers.ts

```diff
@@ -201,6 +201,7 @@ export const getHostEndpoint = async (
       : {};
 
     return {
+      hostInfo: endpointData,
       endpointPolicy: endpointData.metadata.Endpoint.policy.applied.name,
       policyStatus: endpointData.metadata.Endpoint.policy.applied.status,
       sensorVersion: endpointData.metadata.agent.version,
```

Comment:
If we put everything into the `hostInfo` entry, why do we need to have the other 3? Looks like we send redundant data.

Reply:
@semd - True. I was waiting for a response from the 'security-threat-hunting' on whether we needed to maintain backwards compatibility in this API. Looks like Pablo replied today and based on his response, I plan to remove the extra data here in a subsequent PR (don't want to increase the scope of this one)